ide/provisioning/install-debian-packages.sh

```diff
@@ -13,9 +13,10 @@ echo Installing base dependencies...
 apt-get install $APTOPTS make \
   librsvg2-bin xsltproc \
   imagemagick gettext ffmpeg \
-  git quilt zip \
+  git quilt unzip zip \
```

Comment:
What for?

Reply:
`ide/provisioning/install-android-tools.sh` uses `unzip` to extract the Android NDK files. Moved into a separate commit.